### PR TITLE
Rewrite to gradle

### PR DESCRIPTION
**Pull request by michal-josef-spacek**
Based on @galuszkak PR (https://github.com/yarl/vicuna/pull/124)

### CODE REVIEW

**Comment by Nikerabbit** [comment]
Please in the future give @translatewiki heads up about changes affecting translations.

**Comment by michal-josef-spacek** [comment]
@Nikerabbit Ok, I understand your point. Sorry. But I don't know what this mean practically. Only adding @translatewiki to PR or what?

**Comment by Nikerabbit** [comment]
Yes, pinging @translatewiki with a note what you are changing is enough, thanks.

File: README.md [hunk 1]
```diff
@@ -2,39 +2,32 @@
 _ultimate tool for upload files to Wikimedia Commons and other Wikimedia projects_
 
 ## Usage
-For manual, take a look at [project wiki](https://github.com/yarl/vicuna/wiki) and [website](http://yarl.github.io/vicuna).
+For manual, take a look at [project wiki](https://github.com/yarl/vicuna/wiki) and [website](https://yarl.github.io/vicuna).
 
 ## Build and Run
-Program is being written using [NetBeans IDE](https://netbeans.org/) and [Apache Ant](https://ant.apache.org/) is used for building. In order to download and build source code, do following:
+Program is being written using any IDE (NetBeans preffered) you want and [Gradle](https://gradle.org/) is used for building. In order to download and build source code, do following:
```
Comment (ritschwumm) [inline README.md:8]: there's a typo, should be "preferred"

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,6 @@
+all:
```
Comment (ritschwumm) [inline Makefile:1]: what's this file good for?

File: README.md (hunk 1 above)
Comment (michal-josef-spacek) [reply README.md:8]: Fixed in https://github.com/yarl/vicuna/pull/128

File: Makefile (hunk 2 above)
Comment (michal-josef-spacek) [reply Makefile:1]: Removed in https://github.com/yarl/vicuna/pull/128

**Pull request merged**